my.css

```diff
@@ -2,12 +2,10 @@ body {
   margin: 0;
   padding: 0;
 }
-
-.header, .collection, {
-	z-index: 2;
-	position: relative;
-	color: white;
+.wraper, .main {
+	width: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5608084)